python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/experimental/conftest.py

```diff
@@ -343,7 +345,7 @@ def get_sample_schema_config_for_connector(table_name: str) -> Mapping[str, Any]
                                     "is_primary_key": True,
                                 },
                                 "property2": {
-                                    "name_in_destination": "column_name_in_destination_1",
+                                    "name_in_destination": "column_name_in_destination_2",
```

Comment:
There was an error in the example. Fixing to properly test all assets in new column metadata test.